src/component/felles/base-modal/BaseModal.tsx

```diff
@@ -15,27 +15,34 @@ interface BaseModalProps {
 
 export const BaseModal = (props: BaseModalProps) => {
 	const { tittel, open, children, onClose, contentClassName } = props
+	const isOpen = open ?? true
 
 	useEffect(() => {
 		//For å sikre at skjermlesere ikke leser opp innholdet bak modal når den er åpen.
 		Modal.setAppElement('#root')
-	}, [])
+
+		if (isOpen) {
```

Comment:
Bør ikke denne også håndtere hvis isOpen=false? Eller det funker kanskje ikke siden vi har conditional rendering?

Reply:
ja, men kan godt ha det med for sikkerhetsskyld